wee_slack.py

```diff
@@ -419,6 +421,8 @@ def create_buffer(self):
                 w.buffer_set(self.channel_buffer, "localvar_set_type", 'private')
             else:
                 w.buffer_set(self.channel_buffer, "localvar_set_type", 'channel')
+            w.buffer_set(self.channel_buffer, "localvar_set_channel", self.name)
+            w.buffer_set(self.channel_buffer, "localvar_set_server", self.server.team)
```

Comment:
]] Trygve Aaberge 

> In wee_slack.py:
> 
> > @@ -419,6 +421,8 @@ def create_buffer(self):
> >                  w.buffer_set(self.channel_buffer, "localvar_set_type", 'private')
> >              else:
> >                  w.buffer_set(self.channel_buffer, "localvar_set_type", 'channel')
> > +            w.buffer_set(self.channel_buffer, "localvar_set_channel", self.name)
> > +            w.buffer_set(self.channel_buffer, "localvar_set_server", self.server.team)
> 
> Should this take the server alias into account?

Yes, I think so makes sense.

-- 
Tollef Fog Heen
UNIX is user friendly, it's just picky about who its friends are